docs/docs/workflows/storage.md

```diff
@@ -36,6 +36,11 @@ This includes support for [volume snapshots](https://cloud.google.com/kubernetes
 You can use them to bring a volume back to a prior state or provision new volumes.
 Follow the instructions on how to [install the Constellation CSI driver](#installation) or check out the [repository](https://github.com/edgelesssys/constellation-gcp-compute-persistent-disk-csi-driver) for information about the configuration.
 
+</tabItem>
+<tabItem value="aws" label="AWS">
+
+TODO: Is this already supported? Test or write note here!
```

Comment:
@daniel-weisse should know more about this :)

Reply:
Added disclaimer.